Module4.3_Search_and_Sorting_Algorithms/activities/Act3_BinaryHeaps/3.md

```diff
@@ -12,18 +12,42 @@ With the `insert` method properly defined, we can now look at the `delMin` metho
 
 
 
-In order to maintain the heap order property, all we need to do is swap the root with its smallest child less than the root. After the initial swap, we may repeat the swapping process with a node and its children until the node is swapped into a position on the tree where it is already less than both children. The code for percolating a node down the tree is found in the `percDown` and `minChild` methods as seen below:
+In order to maintain the heap order property, all we need to do is swap the root with its smallest child less than the root. We use a while loop to check that we aren't indexing a greater value than the size of the heap.
```

Comment:
can you generalize this more? "root" is a little too specific. This might be remedied by introducing what percolating down is (which is the opposite of percolating up)